AuthorYYMMDD_fignr_001.yaml

```diff
@@ -0,0 +1,74 @@
+# This file can exist with the "source" information only, the entry data exists should be marked as "False" 
+# to indcate that neither the metadata nor the actual data were digitized or provided.
+# use "null" to idicate that a value is not available in the literature
+version: 1 # Is this necessary. Isn't this anyway given by git versioning
+
+source: 
+    doi: https://doi.org/10.1016/0039-6028(85)90985-9
+    bib id: Beden1985 # First Author and YYYY. Cross check for duplicats in the bib file.
+    figure: 2b # depending on the subfigure labels use 1, 1b, or 1_3
+    curve identifier: First cycle # use a unique short description to identify the curve in the figure
+    data exists: True
+    
+curator:
+    name: Albert Engstfeld
+    orcid: https://orcid.org/0000-0002-9686-3948
+    date digitized: 2021.06.01 # YYYY.MM.DD
+    
+data description:
+    type: d # d=digitized, o=original, r=raw
+    measurement type: CV # CV, SCV, LSV see documentation
+    scan rate: 
+        value: 50
+        unit: mV/s # V/s, mV/s
+    potential scale:
+        unit: V # V, mV
+        reference: RHE # given in the plot and not in the experimental section: MSE-3M, MSE-1M, RHE, AgCl2
+    current:        
+        unit: mA # uA, mA, A
+        normalized to: cm-2 # cm-2, m-2, null
+    additional methods: null # FTIR, SXRD, Ring Disc - indicate if any other operando methods were employed
+    links: null #generated automatically
+    comment: # Add whatever you think is important to understand the data
+
+electrochemical system:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: NaCl # replace name by something generic, i.e., for a complex organic compound
+              sum formula:  NaCl
+              concentration: 0.1
+              concentration Unit: M #[M, mM, µM, g kg-1, ...]

```

Comment:
The casing of `unit` is inconsistent with the `unit` above. For all these, we could also just consider to write `concentration: 0.1M`, `scan rate: 50mV/s`.

Reply:
Does it make sense to have it more hierachical?
```
physical quantity:
  value: <float>
  unit: <string>
```

Reply:
Yes, sounds goo to me.